edge-apps/countdown-timer/screenly.yml

```diff
@@ -0,0 +1,44 @@
+---
+user_version: v1.0.0
+description: Screenly Countdown Edge App
+icon: https://playground.srly.io/edge-apps/clock/static/images/icon.svg
+author: Screenly, Inc.
+entrypoint: index.html
+settings:
+  enable_analytics:
+    type: string
+    default_value: 'true'
+    title: Enable Analytics
+    optional: true
+    help_text: Whether to disable Sentry and Google Analytics integrations
+  override_locale:
+    type: string
+    default_value: 'en'
+    title: Override Locale
+    optional: true
+    help_text: Specify a supported locale value (e.g., en_US).
+  override_timezone:
+    type: string
+    default_value: ''
+    title: Override Timezone
+    optional: true
+    help_text: Specify a supported timezone value (e.g., Europe/London).
+  tag_manager_id:
+    type: string
+    default_value: GTM-P98SPZ9Z
+    title: Tag Manager Id
+    optional: true
+    help_text: Specify a Google Tag Manager ID
+  timer:
```

Comment:
Let's rename this variable. It's not clear that this is the timestamp it is counting down to.

Reply:
Agree. Changed to `countdown_timer`. Is that ok?

Reply:
ChatGPT suggests `countdown_to`, `countdown_end_time` or `taget_timestamp`. These are more descriptive of what the value actually is.

Reply:
Thanks for the suggestion, Selected `target_timestamp` as variable name.